libs/ui/src/election_info_bar.test.tsx

```diff
@@ -16,30 +17,51 @@ test('Renders ElectionInfoBar without precinct information by default', () => {
       mode="admin"
     />
   );
+  screen.getByText('General Election');
+  screen.getByText('Nov 3, 2020');
+  screen.getByText('Franklin County,');
+  screen.getByText('State of Hamilton');
+  screen.getByText(hasTextAcrossElements('Software VersionDEV'));
+  screen.getByText(hasTextAcrossElements('Machine ID0000'));
```

Comment:
Maybe we should add a space in here to increase developer readability?

Reply:
Replacing it with this, which is a bit more readable: 
```js
  const versionLabel = screen.getByText('Software Version');
  expect(versionLabel.parentElement?.lastChild).toHaveTextContent('DEV');

  const machineIdLabel = screen.getByText('Machine ID');
  expect(machineIdLabel.parentElement?.lastChild).toHaveTextContent('0000');

  const electionIdLabel = screen.getByText('Election ID');
  expect(electionIdLabel.parentElement?.lastChild).toHaveTextContent(/^.{10}$/);
  ```